portal/src/views/comments/Comments.vue

```diff
@@ -587,8 +602,8 @@ header {
         }
     }
 
-    &.highlight > div {
-        background-color: #a0dbe1;
+    &.highlight {
+        background-color: #f5fbfc;
```

Comment:
I'd create a scss variable and css3 variable 'color-highlight' since there's a big chance it will be reused

Reply:
A good idea. I'm personally really bad at looking for css variables to reuse and need to be better